sample-app/src/components/VisualAutocomplete/VisualSearchBar.tsx

```diff
@@ -29,6 +32,8 @@ interface Props {
   // The debouncing time, in milliseconds, for making API requests for entity previews
   entityPreviewsDebouncingTime: number,
   renderEntityPreviews?: RenderEntityPreviews,
+  hideRecentSearches?: boolean,
```

Comment:
nit: I have a slight preference for naming boolean flags without using negation. In that case, this would be flipped to 'showRecentSearches', and by default the value would be true. I think it's a tad easier to reason about

Reply:
this is what the spec have, but I can ask rose about it